cherrypy/_cpreqbody.py

```diff
@@ -616,11 +613,11 @@ def read_headers(cls, fp):
 
             if line[0] in ntob(' \t'):
                 # It's a continuation line.
-                v = line.strip().decode('ISO-8859-1')
+                v = line.strip().decode(charset, 'replace')
```

Comment:
This looks dangerous to me - using the 'replace' directive in decode is lenient, but almost never what the user intended.